libs/utils/src/lib/services/dot-asset.service.ts

```diff
@@ -0,0 +1,76 @@
+import {
+    DotCMSContentlet,
+    DotCMSTempFile,
+    DotAssetCreateOptions,
+    DotHttpErrorResponse
+} from '@dotcms/dotcms-models';
+import { fallbackErrorMessages } from './dot-temp-file.service';
+
+/**
+ * Create DotAssets based on options passed in DotAssetCreateOptions
+ * @param  {DotAssetCreateOptions} options
+ *
+ */
+export function createDotAsset(
```

Comment:
Are we using this?

Reply:
not for now. In a different PR we need the `dotcms-webcomponents` to start using this. https://github.com/dotcms/core-web/issues/1800